neuronpp/utils/network_status_graph.py

```diff
@@ -81,11 +86,13 @@ def _get_edges(self, weight_name):
             except ValueError:
                 x_pos = self.population_names.index(pop_name)
 
-            y_pos = int(split_name[-1][:-1])
+            # shift down the layer by half its size to vertically center graph
+            y_pos = int(split_name[-1][:-1]) - self.population_sizes[pop_name] // 2
 
             if 'inh' in c.name:
                 self.colors.append('red')
-                y_pos -= 5
+                # todo center vertically by half width of the hid layer
+                y_pos -= 6
```

Comment:
This was original idea how to present plot but now I think it was bad idea to manually shift it like this. But it maybe as it is

Reply:
Problem w linii 95 z rzeczywistym przesunięciem polega na znalezieniu nazwy warstwy poprzedzającej. Jeśli jest nią 'hid_1' to wtedy może być przez
```suggestion
                y_pos -= self.population_sizes['hid_1'] // 2 + self.population_sizes[pop_name]
```